addons/spreadsheet/static/src/data_sources/data_source.js

```diff
@@ -153,17 +153,16 @@ export const LOADING_ERROR = new LoadingDataError();
 export class ModelNotFoundError extends Error {}
 
 /**
- * Perform a `fields_get` on the given model and return the fields.
+ * Return the fields of a given model.
  * If the model is not found, a `ModelNotFoundError` is thrown.
  *
- * @param {ServerData} serverData
+ * @param {object} fieldService
  * @param {string} model
  * @returns {Promise<import("@spreadsheet").OdooFields>}
  */
-export async function getFields(serverData, model) {
+export async function getFields(fieldService, model) {
```

Comment:
I cannot help but to think of the "Field service" app when reading this :sweat_smile: